snowpack/src/scan-imports.ts

```diff
@@ -14,6 +14,9 @@ import {
   readFile,
   SVELTE_VUE_REGEX,
 } from './util';
+import PQueue from 'p-queue';
+
+const CONCURRENT_WORKERS = require('os').cpus().length;
```

Comment:
I think we can bump this up quite a bit, Node.js is great at juggling I/O (vs. compute). I'd rename to:

```
const CONCURRENT_FILE_READS = 1000;
```

Reply:
OK, perfect. Done.